pkg/placement/raft/server.go

```diff
@@ -212,17 +213,35 @@ func (s *Server) StartRaft(ctx context.Context, sec security.Handler, config *ra
 	// Setup Raft configuration.
 	if config == nil {
 		// Set default configuration for raft
-		s.config = &raft.Config{
-			ProtocolVersion:    raft.ProtocolVersionMax,
-			HeartbeatTimeout:   1000 * time.Millisecond,
-			ElectionTimeout:    1000 * time.Millisecond,
-			CommitTimeout:      50 * time.Millisecond,
-			MaxAppendEntries:   64,
-			ShutdownOnRemove:   true,
-			TrailingLogs:       10240,
-			SnapshotInterval:   120 * time.Second,
-			SnapshotThreshold:  8192,
-			LeaderLeaseTimeout: 500 * time.Millisecond,
+		if os.Getenv("PLACEMENT_RAFT_SETTINGS") == "v2" {
```

Comment:
Do we still need this given that the bug was unrelated?

Also, this makes v1 the default, do we want to default to v2?

Reply:
The goal for this change is to reduce the change in actor leadership. It is part of the solution. I made v2 as the default via the helm chart. I can make it the default here instead of people prefer.

Reply:
Default to v2 in code now.